_posts/2020-06-30-not-only-the-code-benefits-from-the-open-source-community.md

```diff
@@ -0,0 +1,49 @@
+
+After this situation was articulated by Lukáš Zapletal, Principal Software Engineer for Satellite/Foreman, we decided it was time to right this wrong. Over the past year, the Satellite documentation team together with enthusiasts from the Foreman community created what Lukáš referred to as [Foreman new-generation documentation](https://community.theforeman.org/t/foreman-new-generation-documentation/15696). This involved migrating Red Hat Satellite guides and adapting them for use both upstream and downstream by working through the following phases:
+
+1. The writing team and the engineers worked together to put an infrastructure in place that would serve both upstream and downstream documentation. Proof of concepts was made and the wider Foreman community was consulted about whether this would be advantageous to them.
+2.  The writers worked with Foreman engineers to review guides to identify pieces that are only Satellite-related and therefore must not be displayed for Foreman users.
+3. The team identified terminology that needed to change between the upstream and downstream and used variables to handle this.
+4. Add notes for items that require an additional explanation for Foreman users, for example, that some procedures work only with the Katello plug-in installed.
+5. The writing team collaborated with the wider community to make the Installation guide work for Debian. You can view the result in [Installing Foreman on Debian/Ubuntu](http://docs.theforeman.org/guides/build/doc-Installing_Server_on_Debian/index-foreman-deb.html).
+
+The vast majority of guides are now [available](http://docs.theforeman.org/web/) in the upstream, but there is still work to be done to consolidate Foreman manuals and ensure that the documentation provides the best possible experience both upstream and downstream. Although the work is still in progress, Foreman community users have provided feedback, for example in [Problem with Foreman 2.1rc2 with Remote Execution on CentOS 8](https://community.theforeman.org/t/problem-with-foreman-2-1rc2-with-remote-execution-on-centos-8/19084/33):
+
+> “The guides on these pages greatly assisted in my understanding of the system…The “Provisioning Guide” was well written, and showed off the [very cool IMHO] hammer(1) CLI interface.”
+
+From a Foreman perspective, we believe that the new Foreman documentation solves some of the problems around lack of focus on user scenarios of the traditional Foreman manuals and implements a better user experience. As the writing team is moving upstream, this also ensures that future content is reviewed by technical writers and therefore has a higher quality.
+
```

Comment:
I would add a call for ideas about how to better integrate the docs. Any concerns about this approach to documentation? How can we progress with this and make docs.theforeman.org a unified one stop shop for documentation for Foreman?